data/json/items/comestibles/mutagen.json

```diff
@@ -176,7 +181,7 @@
     "copy-from": "iv_mutagen_flavor",
     "type": "COMESTIBLE",
     "name": "plant serum",
-    "description": "A super-concentrated mutagen strongly resembling tree sap.  You need a syringe to inject it...  if you really want to?",
+    "description": "A super-concentrated mutagen resembling tree sap.  You need a syringe to inject it...  if you really want to?",
```

Comment:
Hmm, I'm pretty sure that in it's fluid form tree sap is colorless, it's only when it thickens and coagulates to seal a wound that it takes a color.  Still, this is no more unrealistic than any of these other descriptions really.

Reply:
I suppose that's true. I referenced amber in another of the descriptions, so I'll just change it to something else for variety's sake.